packages/protocol-kit/src/contracts/utils.ts

```diff
@@ -23,7 +23,7 @@ export const PREDETERMINED_SALT_NONCE =
 export interface PredictSafeProps {
```

Comment:
Renaming this type would help to differentiate it more from `PredictedSafeProps`. Also the name of the funciton is `predictedSafeAddress()`, so I think it makes sense.
```suggestion
export interface PredictSafeAddressProps {
```

Reply:
done!